package thatline.localup.auth.exception

import thatline.localup.common.exception.BaseException

class InvalidCredentialsException : BaseException("INVALID_CREDENTIALS")
